go/base/context.go

```diff
@@ -340,10 +339,10 @@ func (this *MigrationContext) HasMigrationRange() bool {
 
 func (this *MigrationContext) SetCutOverLockTimeoutSeconds(timeoutSeconds int64) error {
 	if timeoutSeconds < 1 {
-		return fmt.Errorf("Minimal timeout is 1sec. Timeout remains at %d", this.CutOverLockTimeoutSeconds)
+		return fmt.Errorf("Minimal timeout is 1sec.")
```

Comment:
Let's clarify in this message _which timeout_, namely, specify `-cut-over-lock-timeout-seconds`